src/core_plugins/kibana/public/management/sections/settings/advanced_settings.test.js

```diff
@@ -44,6 +45,7 @@ const config = {
   set: () => {},
   remove: () => {},
   isCustom: (setting) => setting.isCustom,
+  isControlledByServer: (key) => key.startsWith('test:overridden'),
```

Comment:
Maybe references to `overridden` should be replaced by `isControlledByServer` here?